File: Leetcode/1984/1984.cpp
class Solution {
public:
    int minimumDifference(vector<int>& nums, int k) {
        sort(nums.begin(),nums.end());
        int n = (int)nums.size();
        int minVal = numeric_limits<int>::max();
        for (int i = 0; i + k - 1 < n; ++i) {
            minVal = min(nums[i+k-1]-nums[i],minVal);
        }
        return minVal;
    }
};

File: Leetcode/1984/1984.go
func min(a, b int) int {
	if a > b {
		return b
	}
	return a
}

func minimumDifference(nums []int, k int) int {
	sort.Ints(nums)
	ans := math.MaxInt32
	for i := 0; i + k - 1 < len(nums); i++ {
		ans = min(ans, nums[i+k-1]-nums[i])
	}
	return ans
}